students/itsabhi/Lesson6/test_mailroom.py

```diff
@@ -0,0 +1,45 @@
+import mailroom4v1
+import unittest
+from unittest.mock import patch
+import os
+
+
+class DonationsTestCases(unittest.TestCase):
+
+    def setUp(self):
+    #Invoked before each test method
+        self.test_donors_dict = {
+        'A G': [10, 20, 30],
+        'N D': [40, 50]
+    }
+    def test_list_donors(self):
+    #Test donor list when the option "list" is selected under the Sending Thanks menu
+        result = mailroom4v1.list_of_donors(self.test_donors_dict)
+        self.assertEqual(list(self.test_donors_dict.keys()), result)  # Convert to list and assert against result
+
+    @patch('mailroom4v1.input', side_effect = ['PJ Mask', 200])
+    def test_sending_thanks_add_donor(self, test_donors_dict):
```

Comment:
The extra parameter here is for the patched input function and should probably be named something like `mock_input`. See this for more details on how you might want to use this parameter: https://stackoverflow.com/a/16053625